src/test/resources/emails/sessionOpeningEmailForInstructor.html

```diff
@@ -9,7 +9,7 @@
 <br>&nbsp;&nbsp; Feedback Session Name: First feedback session
 <br>&nbsp;&nbsp; Deadline: Fri, 30 Apr 2027, 11:59 PM
 <br>
-<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="{The student's unique submission url appears here}">{The student's unique submission url appears here}</a>
+<strong>To submit feedback for the above session, please go to this Web address: </strong><a href="${app.url}/page/instructorFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session">${app.url}/page/instructorFeedbackSubmissionEditPage?courseid=idOfTypicalCourse1&fsname=First+feedback+session</a>
```

Comment:
Perhaps we can add the separator as discussed is the issue thread, that will make this "FYI" email look different from other emails.

Something like this:
~~~
Dear instructor,

The following email has been sent...

=== Email message as seen by the students ===
...
~~~